ext/standard/tests/file/file_put_contents_5gb.phpt

```diff
@@ -0,0 +1,63 @@
+--TEST--
+Test file_put_contents() function with 5GB string
+--SKIPIF--
+<?php
+if (getenv('SKIP_SLOW_TESTS')) {
+    die('skip slow test');
+}
+function get_system_memory(): int|float|false
```

Comment:
NIT: I don't think this can be float...

Reply:
i think it can. but i also think it's missing a SKIPIF: a 32bit PHP running on a 64bit system with over 2GB free ram will overflow 32bit PHP's INT_MAX and return float

but it should also skip 32bit builds because you can't have 5GB worth of data in RAM on 32bit builds.. i think

Reply:
added SKIPIF